lib/plottr_components/src/components/dashboard/files/NewFiles.js

```diff
@@ -78,15 +78,20 @@ const NewFilesConnector = (connector) => {
           </Col>
           {os() == 'unknown' ? null : (
             <Col xs={3}>
-              <div
-                className={cx('dashboard__new-files__item icon', {
+              <Dropdown
+                className={cx('dashboard__new-files__item icon pro-import', {
                   active: activeView == 'import',
                 })}
-                onClick={doImport}
               >
-                <FaRegSnowflake />
-                <div>{t('Import Snowflake Pro')}</div>
-              </div>
+                <Dropdown.Toggle noCaret>
+                  <FaRegSnowflake />
+                  <div>{t('PRO Import')}</div>
```

Comment:
I think this should just be 'Import' because it'll be on non-pro too.
@cameronsutter, can you please comment here?